memfs/memory.go

```diff
@@ -375,3 +391,16 @@ func isReadOnly(flag int) bool {
 func isWriteOnly(flag int) bool {
 	return flag&os.O_WRONLY != 0
 }
+
+func isInDir(dir, path string) bool {
+	p, err := filepath.Rel(dir, path)
+	if err != nil {
+		return false
```

Comment:
I think the only way to reproduce that (from public methods) is using Windows paths :s